frontend-nextjs/features/readme.md

```
@@ -0,0 +1,2 @@
+特定のページに関するコンポーネントを置く (auth/LoginForm.tsx, notebook/NotebookList.tsxなど
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)